doc/release-notes/7000-mpconfig-support.md

```diff
@@ -0,0 +1,8 @@
+# Broader MicroProfile Config Support for Developers
+
+As of this release, many [JVM options](https://guides.dataverse.org/en/latest/installation/config.html#jvm-options)
+can be set using any [MicroProfile Config Source](https://docs.payara.fish/community/docs/Technical%20Documentation/MicroProfile/Config/Overview.html#config-sources).
+
+Currently this change is only relevant to developers but as settings are migrated to the new "lookup" pattern documented in the [Consuming Configuration](https://guides.dataverse.org/en/latest/developers/configuration.html) section of the Developer Guide, anyone installing the Dataverse software will have much greater flexibility when configuring those settings, especially within containers. These changes will be announced in future releases.
+
+Please note that an upgrade to Payara 5.2022.3 or higher may be required. Payara 5.2021.5 threw exceptions, as explained in PR #8823.
```

Comment:
I looked at the stack trace and went looking at [PayaraConfigL228](https://github.com/payara/Payara/blob/e3bee021061f7ded022b92419a1f92e0780d7e50/nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/spi/PayaraConfig.java#L228) as seen in the log.

Guess what, this was fixed with payara/Payara#5431 and released with [Payara 5.2021.8](https://github.com/payara/Payara/releases/tag/payara-server-5.2021.8).

Should we add that detail?

```suggestion
Please note that an upgrade to Payara 5.2021.8 or higher is required to make use of this. Payara 5.2021.5 threw exceptions, as explained in PR #8823.
```

Reply:
Sure! Committed! Thanks!